dartsim/src/EntityManagementFeatures.cc

```diff
@@ -393,22 +400,19 @@ const std::string &EntityManagementFeatures::GetLinkName(
 std::size_t EntityManagementFeatures::GetLinkIndex(
```

Comment:
I didn't see any tests written that use `EntityManagementFeatures::GetLinkIndex`, would it be worth adding some in?

Reply:
Added tests in https://github.com/ignitionrobotics/ign-physics/pull/227/commits/0bb7d9e9d787d815f7fd3db15aa2da9a815c40b2